### PR TITLE
Move td-reproducible-tool to td-shim-tool directory

### LINKED ISSUE #289: need move td-reproducible-tool to td-shim-tool dir

### PR DESCRIPTION
**Pull request by liuw1**
Move td-reproducible-tool to td-shim-tool directory
Fix #289

### CODE REVIEW

File: td-shim-tools/Cargo.toml [hunk 1]
```diff
@@ -23,8 +23,14 @@ required-features = ["signer"]
 name = "td-shim-checker"
 required-features = ["loader"]
 
+[[bin]]
+name = "td-reproducible-tool"
```
Comment (jiangliu) [inline td-shim-tools/Cargo.toml:27]: Should this be renamed to "td-shim-xxxx"?

Comment (jyao1) [reply td-shim-tools/Cargo.toml:27]: I am OK with that.
Do you have a proposal? "td-shim-strip-info" ?

**Comment by jyao1** [comment]
Why you mix the unit test in event log? Please remove it.

**Comment by liuw1** [comment]
> Why you mix the unit test in event log? Please remove it.

Done

**Comment by jyao1** [comment]
please resolve conflict.

**Review by jyao1 — APPROVED** [review]

**Pull request merged**